packages/fether-react/src/utils/testHelpers/mock.js

```diff
@@ -50,9 +50,18 @@ export const post$ = {
   })
 };
 
```

Comment:
suggest also modifyng the value of the erc20.address on Line 26 to be `'THIBCoin'` instead of `'foo'` (otherwise there is too much foo :) and tests become more of a chore to read )